### ISSUE TITLE
some help on compiling updated binary for Linux

### ISSUE THREAD

**Issue by rcampbel3** [PadWorld-Entertainment/wop-engine#4]
I'm on linux and I compile ioquake3 source pretty regularly. What I'm looking for is a simple explanation of what config options I need to use to build a replacement binary for linux for this world of padman source repo -- the Makefile currently builds an ioquake3 binary with all the ioquake3 defaults. I can probably figure it out on my own eventually, but... if someone has a list of exactly what one needs to do to build and updated binary with the  from source for each platform... that would be super helpful. Also... can I use the latest ioquake3 source so I can use seta cl_renderer "opengl2"?

here's a start:

<pre>
dotfile dir in ioquake3: ~/.q3a
dotfile dir in wop: ~/.padman

main config + extras directory in ioquake3: ~/.q3a/baseq3
main config + extras directory in wop: ~/.padman/wop

main config file in ioquake3: ~/.q3a/baseq3/q3config.cfg
main config file in wop: ~/.padman/wop/wopconfig.cfg

binary name ioquake3: ioquake3.x86_64
binary name wop: wop.x86_64

</pre>

so, here's my guess so far:

make BUILD_STANDALONE=1 CLIENTBIN=wop BASEGAME=wop

Thanks for any help, and thanks for all the fun!

**Comment by robo9k** [comment]
I do not quite understand the question. Which branch are you trying to build, `master`? You should probably use branch `1.6.0` currently.

You can take a look at e.g. 3eaff7fc290d5d671a8b40afac65966e132ef90b from the `1.7-dev` branch where @codemasher started a "rebase" of the old code onto more recent ioq3 master. Unfortunately the changes for a branded standalone game are scattered across a few code and config files as well as logos and such for distribution packages.

In theory you can use a vanilla ioq3 engine build to run WoP content, setting a few configuration variables and startup switches with minor functionality loss. I used to have a bash script for that on my dusty Linux box, so if that's something you want to do I can try to dig it up again 😄

**Comment by rcampbel3** [comment]
Hi, Yes... I was trying to build master, and I do build ioq3 from master regularly. It seems that there should be a project-specific README.md or something that gets people started on the right path instead of the default ioq3 readmes... which currently I *ASSUME* is to build a binary that is drop-in compatible with the padman binary downloads. based on a 1.2 source code zip inside of my padman install dir and some quick grepping and diffing, I did find some values in headers that I needed to change that got me about 70% of the way there, but I felt like I'm guessing and reverse engineering.

Thanks for the link to the changes in 1.7-dev branch - that's most of what I was looking for. If you find your script from your dusty Linux box, my dusty linux boxes would appreciate it.
